src/main/resources/application.yml

```diff
@@ -0,0 +1,65 @@
+spring:
+  datasource:
+    url:  jdbc:mysql://localhost:3306/vetlog
+    username:  ${USER}
+    password:  ${PASSWORD}
+    driverClassName: com.mysql.jdbc.Driver
+    maxActive:  50
+    maxIdle: 5
+    minIdle: 25
+    initialSize: 10
+    validationQuery: SELECT 1
+    testOnBorrow: true
+    testOnReturn: false
+    testWhileIdle: true
+    timeBetweenEvictionRunsMillis: 5000
+    minEvictableIdleTimeMillis: 60000
+    maxWait: 10000
+    template:
+    cache: false
+  jpa:
+    generateDdl: true
+    hibernate:
+      DdlAuto: update
+  profiles:
+    include: development
+  messages:
+    basename: i18n/messages
+  servlet:
+    multipart:
+      maxFileSize: 10MB
+      maxRequestSize: 10MB
+  cloud:
+    gcp:
+      projectId: vetlog-357216
+      core:
+        enabled: true
+      credentials:
+        location: classpath:vetlog.json
+  flyway:
+    baseline-on-migrate: true
+message: Hello Vetlog!
+jmailer:
+  url: https://jmailer.josdem.io/emailer/
+  path: message
+baseUrl: https://vetlog.org/
+template:
+  register:
+    name: register.ftl
+    path: recovery/activate/
+  forgot:
+    name: forgotPassword.ftl
+    path: recovery/forgot/
+  adoption:
+    name: adoption.ftl
+breedsByTypeUrl: https://vetlog.org/breed/list
+imageBucket: vetlog-bucket
+prescriptionBucket: prescription-bucket
+gcpUrl: https://storage.googleapis.com/
+defaultImage: default-image.png
+token: userToken
```

Comment:
haven't checked the usage, but usually all tokens should also be hidden

Reply:
Do not worry, the real token is on the server `systemctl` configuration.